assets/src/scripts/utils.js

```diff
@@ -146,25 +146,25 @@ export const mediaQuery = function (minWidth) {
 /**
  * Calculate the start time of a time range based on a time-delta string and the end time
  *
- * @param {String} period -- ISO duration for date range of the time series
+ * @param {String} period -- ISO duration for date range of the time series - which has the form of something like
+ * P7D in which the 'P' stands for 'period', the '7' is units in the period, and 'D' is the type of unit
+ * in this case 'Days.' There are only two unit types here, 'Days' or 'Years'
  * @param {Number} endTime -- the end time as universal time
  * @param {String} ianaTimeZone -- Internet Assigned Numbers Authority designation for a time zone
  * @returns {Number} the start time as universal time
  */
-export const calcStartTime = function (period, endTime, ianaTimeZone) {
+export const calcStartTime = function(period, endTime, ianaTimeZone) {
```

Comment:
So we only did this before so that we could fetch the "compare" data. I don't think you need to do this for the custom days from today but rather just use the period parameter call when fetching the data. See previous comment in the date-controls.js module.